### ISSUE TITLE
Transport support for existing udfs written in presto for execution in hive/spark etc

### ISSUE THREAD

**Issue by amalakar** [linkedin/transport#9]
Presto seems to support lot of rich functions that we find very useful, examples are: 

- string functions - https://prestodb.github.io/docs/current/functions/string.html
- geospatial function - https://prestodb.github.io/docs/current/functions/geospatial.html
- url functions - https://prestodb.github.io/docs/current/functions/url.html

These are just few of all the functions supported by presto.

If transport also had apis to translate presto functions to transport udf, it could unlock the rich set of presto functions to other execution engines (to spark or flink for example). A distro of presto out-of-the-box functions made available using transport for hive/flink can be of immense value to users. 

Extending this idea, if the same is done to support hive udfs, that would help adoption as teams who already has lot of hive udfs can take advantage of the framework right away without having to re-write those udfs using transport udf.

**Comment by wmoustafa** [comment]
Hi Arup,

Thanks for filing this issue. I think automatic migration of Presto UDFs to Transport can be very tricky, but I am open to ideas. Sounds like a very interesting project. That being said, we are planning on extending the framework with some out-of-the-box UDFs with valuable utility to many data users, such as the ones you mentioned. Please let us know if you happen to write some of them in the Transport API, and we will gladly work with you to check them into the project.

**Comment by amalakar** [comment]
Agreed, this may not be very trivial but could have lot of value if it can be done.

Regarding out-of-the-box transport udfs that would be great. Would definitely like to contribute back if we work on transport udfs.
